k8s/example.yaml

```diff
@@ -0,0 +1,182 @@
+            #
+            # - name: "SPICEDB_GRPC_TLS_KEY_PATH"
+            #   valueFrom:
+            #     secretKeyRef:
+            #       name: "spicedb"
+            #       key: "SPICEDB_GRPC_KEY_PATH"
+            # - name: "SPICEDB_GRPC_TLS_CERT_PATH"
+            #   valueFrom:
+            #     secretKeyRef:
+            #       name: "spicedb"
+            #       key: "SPICEDB_GRPC_CERT_PATH"
+            # - name: "SPICEDB_HTTP_TLS_KEY_PATH"
+            #   valueFrom:
+            #     secretKeyRef:
+            #       name: "spicedb"
+            #       key: "SPICEDB_HTTP_KEY_PATH"
+            # - name: "SPICEDB_GRPC_HTTP_CERT_PATH"
+            #   valueFrom:
+            #     secretKeyRef:
+            #       name: "spicedb"
+            #       key: "SPICEDB_HTTP_CERT_PATH"
+            #
+            # These flags are used to enable a persistent datastore along
+            # with cluster dispatching. For more info see:
+            # https://docs.authzed.com/spicedb/selecting-a-datastore
+            #
+            # - name: "SPICEDB_DATASTORE_ENGINE"
+            #   value: "cockroachdb"
+            # - name: "SPICEDB_DATASTORE_CONN_URI"
+            #   valueFrom:
+            #     secretKeyRef:
+            #       name: "spicedb"
+            #       key: "SPICEDB_DATASTORE_CONN_URI"
+            # - name: "SPICEDB_DISPATCH_UPSTREAM_ADDR"
+            #   value: "kubernetes:///spicedb:dispatch"
+            - name: "SPICEDB_LOG_LEVEL"
+              value: "debug"
+            - name: "SPICEDB_HTTP_ENABLED"
+              value: "true"
+            - name: "SPICEDB_GRPC_SHUTDOWN_GRACE_PERIOD"
+              value: "1s"
+            - name: "SPICEDB_GRPC_PRESHARED_KEY"
+              valueFrom:
+                secretKeyRef:
+                  name: "spicedb"
+                  key: "SPICEDB_GRPC_PRESHARED_KEY"
+          ports:
+            - name: "grpc"
+              containerPort: 50051
+              protocol: "TCP"
+            - name: "http"
+              containerPort: 8443
+              protocol: "TCP"
+            - name: "dispatch"
+              containerPort: 50053
+              protocol: "TCP"
+            - name: "prometheus"
+              containerPort: 9090
+              protocol: "TCP"
+          readinessProbe:
+            exec:
+              command: ["grpc_health_probe", "-v", "-addr=localhost:50051"]
+            failureThreshold: 5
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 5
+---
+apiVersion: "v1"
+kind: "ServiceAccount"
+metadata:
+  name: "spicedb"
+  labels:
+    app.kubernetes.io/name: "spicedb"
+    app.kubernetes.io/version: "example"
+---
+apiVersion: "rbac.authorization.k8s.io/v1"
+kind: "Role"
+metadata:
+  name: "watch-service"
+  labels:
+    app.kubernetes.io/name: "spicedb"
+    app.kubernetes.io/version: "example"
+rules:
+  - apiGroups: [""]
+    resources: ["endpoints"]
+    verbs: ["get", "watch"]
```

Comment:
you should need `list` too?

Reply:
not according to the kuberesolver readme

Reply:
oh, super weird that it directly queries instead of using a client-go informer (which would've used list+watch)